File: LV.1/이상한 문자 만들기/solution.py
def solution(s):
    a= s.split(' ')
    new=''

    for i in a:
        b=len(i)
        for j in range(0,b):
            if j%2 ==0:
                u=i[j].upper()
                new = new + u
            else:
                l = i[j].lower()
                new = new + l
        new = new +" "
    return new[0:len(new)-1]

File: LV.1/이상한 문자 만들기/Solution.java
class Solution {
    public String solution(String s) {
        String answer = "";
        String[] str = s.split(" ",-1);
        
        for (int i=0; i < str.length; i++) {
            for (int j=0; j < str[i].length(); j++) {
                if (j%2 ==0) {
                    str[i] = str[i].toUpperCase();
                    String temp = Character.toString(str[i].charAt(j));
                    answer += temp;
                } else {
                    str[i] = str[i].toLowerCase();
                    String temp = Character.toString(str[i].charAt(j));
                    answer += temp;
                }
            } answer += " ";
        }
        return answer.substring(0,answer.length()-1);
    }
}
